aea/cli/scaffold.py

```diff
@@ -55,6 +55,23 @@
 
 
 @click.group()
+@click.option(
+    "-vendor",
+    default="open_aea",
+    type=str,
+    help="Specify component vendor.",
+)
+@click.option(
+    "-package-dir",
+    default="packages/",
+    type=str,
+    help="Path packages directory.",
+)
+@click.option(
+    "--local-package",
+    is_flag=True,
+    help="Scaffold skill outside a agent directory.",
+)
```

Comment:
To scaffold package outside an AEA project with the default setting run
```bash
aea scaffold --local-package [COMPONENT] [SKILL_ID]
```
By default, it'll try to find the packages directory in the current working directory, and the vendor is set to open_aea. `vendor` and `package_dir` are also configurable.

There's a small issue in the packages directory. There needs to be an __init__.py file in the components folder `packages/component/__init__.py` which is a bit hard to configure if the components directory does not exist already. So for now I'm already assuming that `packages/component` exists and already has an __init__.py file.

Reply:
Since we use default author for the vendor (see above), we end up with `packages/vendor/...` path. Why do we need the init file?